### ISSUE TITLE
Stable Learning Rate for Depth greater than 6?

### ISSUE THREAD

**Issue by afiaka87** [lucidrains/DALLE-pytorch#73]
I'm trying my best to run a 64 depth dalle over here. It barely fits in memory, and I have to use a batch_size of 1 or it OOMs on an RTX 3090, but it does seem to get going.

Unfortunately, the loss drops way too quickly and gets stuck in the wrong local minimum. 

I've had mild success with lowering the learning rate from 3e-4 to 1e-4, but being exhaustive about tuning the learning rate is tough when training takes this many resources. 

If anyone has been messing with more layers/head/depth and has found stable learning rates, let me know.
